AudioKit/Common/Nodes/Playback/Player/AKPlayer.swift

```diff
@@ -162,21 +160,14 @@ public class AKPlayer: AKNode {
     
     public var endTime: Double = 0 {
         didSet {
-            if endTime > duration {
-                endTime = duration
-            }
+            endTime = min(endTime, duration)
         }
     }
     
-    /// Current time of the player in seconds.
+    /// - Returns: Current time of the player in seconds.
+    /// Use setTime() or startTime to set the time
     open var currentTime: Double {
-        get {
```

Comment:
Why remove the setter?

Reply:
There's a lot of redundancy for startTime once AKTiming is added. Seems kind of weird to have setTime() startTime AND currentTime. What do you suggest?

Reply:
I would just remove setTime.  What's startTime for?

Reply:
setTime is part of the AKTiming protocol. One of the reasons I chose not to originally use the protocol.
startTime and endTime allow you to set the in point / out point independent of playback. It's also used to store the last in/out point specified by the play(from: to:)

Reply:
So it is.  That _is_ confusing.

Reply:
So startTime is the same as setTime, correct?